fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -1577,4 +1580,31 @@ public TStorageFormat getStorageFormat() {
         }
         return tableProperty.getStorageFormat();
     }
+
+    public boolean satisfyHashDistribution(List<Expr> hashExprs) {
+        if (hashExprs == null) {
```

Comment:
partitionExprs is null has checked in  `if (partitionExprs == null) partitionExprs = groupingExprs;`